### PR TITLE
adding missing tests

### PR DESCRIPTION
**Pull request by prateekbh**
R: @jeffposnick @addyosmani @gauntface

This PR adds the missing tests for response manager.

### CODE REVIEW

File: packages/sw-background-sync-queue/test/unit/response-manager.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ */
+
+/* eslint-env mocha, browser */
+/* global chai, goog */
+
+'use strict';
+
+describe('response-manager test', () => {
+  const response = 'VALUE';
+  const idbHelper = new goog.backgroundSyncQueue.test.IDBHelper(
```
Comment (gauntface) [inline packages/sw-background-sync-queue/test/unit/response-manager.js]: What is goog.backgroundSyncQueue.test ?

File: packages/sw-background-sync-queue/test/unit/response-manager.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  it('check put', (done) => {
+      resManager.putResponse({
+          hash: 'somehash',
+          idbObject: {},
+          response: new Response(response),
+          idbQDb: idbHelper,
+      }).then(()=>{
+        idbHelper.get('somehash').then((cachedResponse) => {
+            const reader = new window.FileReader();
```
Comment (gauntface) [inline packages/sw-background-sync-queue/test/unit/response-manager.js]: Why does this need to be passed through a FileReader? Is cachedResponse not just a normal Response object?

Comment (prateekbh) [inline packages/sw-background-sync-queue/test/unit/response-manager.js]: I store response body as BLOB in the iDB. Hence FileReader for conversion back

File: packages/sw-background-sync-queue/test/unit/response-manager.js (hunk 1 above)
Comment (prateekbh) [inline packages/sw-background-sync-queue/test/unit/response-manager.js]: `goog.backgroundSyncQueue.test` is the namespace where all the libs are compiled and put for testing separately

File: packages/sw-background-sync-queue/test/unit/response-manager.js (hunk 2 above)
Comment (gauntface) [inline packages/sw-background-sync-queue/test/unit/response-manager.js]: Could you add a comment just to highlight this.

Comment (prateekbh) [inline packages/sw-background-sync-queue/test/unit/response-manager.js]: done 👍

**Comment by gauntface** [comment]
LGTM

**Comment by prateekbh** [comment]
@gauntface @jeffposnick someone needs to approve.. if looks good

**Pull request merged**